model/task_queue_service.go

```diff
@@ -122,382 +116,12 @@ func (s *taskDispatchService) ensureQueue(ctx context.Context, distroID string)
 			return nil, err
 		}
 	default:
-		distroDispatchService = newDistroTaskDispatchService(taskQueue, d.DispatcherSettings.Version, s.ttl)
+		// TODO: Error?
 	}
 
 	s.cachedDispatchers[distroID] = distroDispatchService
 	return distroDispatchService, nil
 }
 
```

Comment:
nit: there's a bunch of leftover new lines at the bottom of the file. Can we delete them?